wdl-lint/tests/lints/between-import-whitespace/source.errors

```diff
@@ -51,3 +51,11 @@ warning[Whitespace]: more than one blank line in a row
    │  
    = fix: remove the unnecessary blank lines
 
+note[DescriptionMissing]: description key missing in meta section
+   ┌─ tests/lints/between-import-whitespace/source.wdl:22:5
+   │
+22 │     meta {}
```

Comment:
Same question here as your last PR: should the reported span encapsulate the whole meta object or just this keyword?

Reply:
I personally think it's find to point at the `meta` keyword to say "this section is missing a `description` key".

Highlighting the whole section feels like it would call attention to mostly valid things.